<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOrigen">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>80</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar carpeta origen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDestino">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar carpeta destino</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbInput">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbOutput">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ORIGEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DESTINO</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
